stock_picking_reorder_lines/README.rst

```diff
@@ -0,0 +1,59 @@
+[![Licence](https://img.shields.io/badge/licence-AGPL--3-blue.svg)](http://www.gnu.org/licenses/agpl-3.0-standalone.html)
```

Comment:
The proper format is

    .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
        :alt: License: AGPL-3